src/vl-rich-data-field.js

```diff
@@ -164,15 +167,21 @@ export class VlRichDataField extends vlElement(HTMLElement) {
     }));
   }
 
+  get __headerContent() {
+    return this.label || (this._labelSlotElement ? this._labelSlotElement.innerHTML : undefined);
```

Comment:
Moet dit niet zijn lege string ipv undefined?

Reply:
Kan ook, maar in mijn ogen is er een verschil tussen een empty string/header en een undefined header. Daarom dat ik hier voor undefined ben gegaan.